get_config.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+repo_dir="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+
+get_debian_vers_from_pkg_repo() {
+    debian_version_line=$(grep -m1 'debian_ref=' "$repo_dir/prepare_source")
+    echo "${debian_version_line#*=}" | tr -d '"'
+    }
+
+get_vers_orig_from_pkg_repo() {
+    version_orig_line=$(grep -m1 'version_orig=' "$repo_dir/prepare_source")
+    echo "${version_orig_line#*=}" | tr -d '"'
+    }
+
+deb_version=$(get_debian_vers_from_pkg_repo)
+version_orig=$(get_vers_orig_from_pkg_repo)
+
+arch=$(dpkg --print-architecture 2>/dev/null)
+
+
+apt-get install -y --no-install-recommends quilt flex bison build-essential gcc-14 make fakeroot libncurses-dev kernel-wedge python3-jinja2 python3-dacite wget gpgv
+
+apt-get remove --purge sqopv -y
+
+mkdir -p /tmp
+
+cd /tmp
+
+git clone https://salsa.debian.org/kernel-team/linux.git
```

Comment:
this repo is large and a full clone is not needed here. this can be way faster if we take only the latest commit of our desired tag/branch using

```bash
git clone --depth=1 --branch=debian/6.12.43-1 https://salsa.debian.org/kernel-team/linux.git
```